docs/reference/gps.md

```diff
@@ -165,6 +168,8 @@ The `gpsReference` field of the [WorldInfo](worldinfo.md) node can be used to de
 
 The `wb_gps_get_speed` function returns the current [GPS](#gps) speed in meters per second.
 
+The `wb_gps_get_speed_vector` function returns the current [GPS](#gps) speed vector (difference between current and previous position) in centimeters.
```

Comment:
We rely on the metric system for units.
Therefore, the unit for the speed vector should be meters per second.